File: variable/main03.c

// 自定义输出数值

//请定义1个整型的变量. 将这个整型的变量的值输出
//要求: 输出的整型的变量的数据如果不足8位 则在这个数据的前面以0补齐8位.
//(提示:%8d可以使整型数据输出位数为8,%08d可以在不足的位置上补0)
#include <stdio.h>

int main(int argc, const char * argv[]) {
    int num;
    num = 3;
    printf("%08d \n",num);
}

File: 宏定义/main03.m

// 将计算圆面积的宏定义存储在一个头文件，输入半径便可得到圆的面积

#import <Foundation/Foundation.h>
#define CE(r) r * r * 3.14
int main(int argc, const char * argv[]) {
    int r = 5;
    int res = CE(r);
    //int circle = r * r * 3.14;
    printf("%d \n",res);
    return 0;
}
